src/components/SkillPage/SkillListing.js

```diff
@@ -844,6 +845,19 @@ class SkillListing extends Component {
                   </Dialog>
                 </div>
               )}
+              <div>
+                <a href="https://skills.susi.ai">
```

Comment:
Use react-router-dom

Reply:
You need to know where the user is coming to the skills from, whether root page or the category page or any other possibility.